app/domain/operations/families/create.rb

```diff
@@ -34,20 +37,31 @@ def create_family(family_params)
         end
       end
 
-      def create_family_params(values)
+      def create_family_params(outbound_account_entity, values)
         Success(
           {
             correlation_id: values[:inbound_family_cv][:hbx_id],
             family_hbx_id: values[:inbound_family_cv][:hbx_id],
             inbound_raw_payload: values[:inbound_family_cv],
             inbound_payload: values[:inbound_family_cv].to_json,
+            outbound_payload: outbound_account_entity.to_json,
             primary_person_hbx_id: @primary_person_hbx_id,
             job_id: values[:job].id,
             inbound_after_updated_at: values[:inbound_after_updated_at]
           }
         )
       end
 
+      def construct_outbound_account_entity(family_cv)
+        inbound_family_entity = ::AcaEntities::Operations::CreateFamily.new.call(
+          family_cv
+        ).success
+        account = ::AcaEntities::Crm::Transformers::FamilyTo::Account.new.call(inbound_family_entity)
+        account.success? ? Success(account.value!) : Failure(account.failure)
```

Comment:
@jacobkagon Can you make these two different steps instead of merging them into one? This way we can handle the failure for CreateFamily operation and also let the yield handle the success or failure moand.